### PR TITLE
Do not assume remote video menu dom structure and change to click

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: src/test/java/org/jitsi/meet/test/MuteTest.java [hunk 1]
```diff
@@ -159,29 +159,25 @@ public void ownerMutesParticipantAndCheck()
 
         WebElement elem = owner.findElement(By.xpath(
             TestUtils.getXPathStringForClassName("//span", "remotevideomenu")
-            + "/i[@class='icon-thumb-menu']"));
+            + "//i[@class='icon-thumb-menu']"));
```
Comment (bbaldino) [inline src/test/java/org/jitsi/meet/test/MuteTest.java]: is it worth having constants for these selectors?

Comment (virtuacoplenny) [inline src/test/java/org/jitsi/meet/test/MuteTest.java]: Yes, it's worthwhile but almost all these files have these selectors inlined. What needs to happen, and what I'd really like to do this year, is start moving these over into page objects to encapsulate these lookups, moving these to constants in the process.

Comment (virtuacoplenny) [inline src/test/java/org/jitsi/meet/test/MuteTest.java]: (In other words, I agree but not sure I should do it at this point for consistency.)

Comment (virtuacoplenny) [inline src/test/java/org/jitsi/meet/test/MuteTest.java]: But then again why not? It is better for readability. I'll just make the change now.

**Pull request merged**